extensions/import/src/test/wizard/pages/fileConfigPage.test.ts

```diff
@@ -11,22 +11,42 @@ import { ImportDataModel } from '../../../wizard/api/models';
 import { TestImportDataModel, TestQueryProvider } from '../../utils.test';
 import { FileConfigPage } from '../../../wizard/pages/fileConfigPage';
 import * as should from 'should';
+import { ImportPage } from '../../../wizard/api/importPage';
+import * as constants from '../../../common/constants';
 
-describe('import extension wizard pages', function () {
+describe('File config page', function () {
 
+	// declaring mock variables
 	let mockFlatFileWizard: TypeMoq.IMock<FlatFileWizard>;
 	let mockApiWrapper: TypeMoq.IMock<ApiWrapper>;
 	let mockImportModel: TypeMoq.IMock<ImportDataModel>;
 
+	// declaring instance variables
+	let fileConfigPage: FileConfigPage;
+	let wizard: azdata.window.Wizard;
+	let page: azdata.window.WizardPage;
+	let pages: Map<number, ImportPage> = new Map<number, ImportPage>();
+
 	this.beforeEach(function () {
+		// initializing mock variables
 		mockApiWrapper = TypeMoq.Mock.ofType(ApiWrapper);
 		mockFlatFileWizard = TypeMoq.Mock.ofType(FlatFileWizard, TypeMoq.MockBehavior.Loose, undefined, TypeMoq.It.isAny(), mockApiWrapper.object);
 		mockImportModel = TypeMoq.Mock.ofType(TestImportDataModel, TypeMoq.MockBehavior.Loose);
+
+		// using the actual vscode and azdata apis.
```

Comment:
If you're just going to be using the normal api's then don't bother mocking it in the first place I would say